tools/shared/pragma.c

```diff
@@ -1297,6 +1299,12 @@ do_sw(void)
      */
     bset(DIR_OFFSET(currdir, x[14]), 8);
     break;
+  case SW_FORCEINLINE:
+    if (no_specified)
+      bclr(DIR_OFFSET(currdir, x[191]), 0x2);
+    else
+      bset(DIR_OFFSET(currdir, x[191]), 0x2);
```

Comment:
If I understand correctly, the pragma that appears later in the code should override a conflicting pragma that appears earlier. `SW_FORCEINLINE` should clear `XBIT(14, 8)`, and `SW_NOINLINE` should clear `XBIT(191, 2)`.

Reply:
Fixed in 752770a